probes/fuzzedWithOSSFuzz/def.yml

```diff
@@ -29,4 +29,16 @@ remediation:
     - Over time, try to add fuzzing for more functionalities of your project.
   markdown:
     - Follow the steps in [https://github.com/google/oss-fuzz](https://github.com/google/oss-fuzz) to integrate fuzzing for your project.
-    - Over time, try to add fuzzing for more functionalities of your project.
\ No newline at end of file
+    - Over time, try to add fuzzing for more functionalities of your project.
+ecosystem:
+  languages:
+    - c
+    - c++
+    - go
+    - java
+    - javascript
+    - python
+    - rust
+    - typescript
+  clients:
+    - oss-fuzz
```

Comment:
whoops missed one. I think this should be github/gitlab, not oss-fuzz. for reasons above

Reply:
https://github.com/ossf/scorecard/pull/3797/commits/7bcf9773f5e16ed35f16f40534b74af860cc72f5